### ISSUE TITLE
Default avatar color are not matching EleWeb

### ISSUE THREAD

**Issue by bmarty** [vector-im/element-android#4676]
|Web|Element Android|
|<img width="335" alt="image" src="https://user-images.githubusercontent.com/3940906/145542982-c4fe6540-14b3-4020-afcb-a5136e810c9b.png">|<img width="161" alt="image" src="https://user-images.githubusercontent.com/3940906/145542892-71f2cab0-ce6e-4982-8913-91c8bebbcbea.png">|

Maybe the algorithm has changed on web...

**Comment by gaelledel** [comment]
Yes we should aim to find the same colours for all platforms. It seems to be a sync issue between platforms. @gsouquet and @SBiOSoftWhare might be able to help?
